docs/admin/troubleshooting.md

```diff
@@ -327,7 +327,7 @@ sudo ipcrm -M 0x10242049
 1. When the server configuration is changed, it's necessary to restart the agent.
 1. `DER_UNREACH(-1006)`: Check the socket ID consistency between PMem and NVMe. First, determine which socket you're using with `daos_server network scan -p all`. e.g., if the interface you're using in the engine section is eth0, find which NUMA Socket it belongs to. Next, determine the disks you can use with this socket by calling `daos_server nvme scan` or `dmg storage scan`. e.g., if eth0 belongs to NUMA Socket 0, use only the disks with 0 in the Socket ID column.
 1. Check the interface used in the server config (`fabric_iface`) also exists in the client and can communicate with the server.
-1. Check the access_points of the agent config points to the correct server host.
+1. Check the `access_points` of the agent config points to the correct server hosts.
```

Comment:
I don't think we want to put those implementation details in the user documentation. In practice, it really doesn't matter which server addresses go in the agent config. If anything, I'd be inclined to change the guidance to just use a hostlist that covers all of the servers.

Reply:
To expand a bit on this comment, as I've noted elsewhere, longer-term I think we want to move away from exposing a fixed set of server addresses as "special" to the agent. Keeping this somewhat vague now gives us flexibility to change the server side of things without requiring changes to the agent config process.

I've pushed the latest commit with `Features: control`. I was under the impression that the test harness was smart enough to automatically test all changed .py files now, but maybe not?

Reply:
It's only smart enough when you modify specific functional tests.
- modify specific functional test -> run that test
- modify ftest/util -> run `pr` - not smart!
- modify dfs -> run `dfs` - slightly smart
- anything else -> run `pr` - not smart!

The mapping is here
https://github.com/daos-stack/daos/blob/master/src/tests/ftest/tags.yaml

This hasn't gotten much attention since I originally added it (I haven't had time), but we could add something like this e.g. Or whatever you think makes sense for the code you regularly edit.
But keep in mind with this mapping, even if you modify a single line comment, all `control` tests will be included. So devs should address this for their feature areas.
```
# Run "control" for files under "src/control/"
# Run "control" and "security" for files under "src/control/security"
src/control/: control
src/control/security/: security
```

The way to override this behavior would be to use `Test-tag: <specific tests I want to run>`
`Test-tag` overrides whatever CI auto-determined.
`Features` is on top of whatever CI auto-determined.